examples/rfq/README.md

```diff
@@ -0,0 +1,278 @@
+export OWNER_1=b4f8586041a07323bd4f4ed2d758bf1b9a977eabfd4c00e2f12d08a0899485fd
+```
+
+The `--extra-wallets 1` option creates an additional user chain and wallet - we will use it for the
+user requesting a quote.
+
+Now we have to publish and create the fungible applications.
+
+```bash
+(cd examples/fungible && cargo build --release --target wasm32-unknown-unknown)
+
+APP_ID_0=$(linera --with-wallet 0 project publish-and-create \
+           examples/fungible \
+           --json-argument '{ "accounts": { "User:'$OWNER_0'": "500", "User:'$OWNER_1'": "500" } }' \
+           --json-parameters "{ \"ticker_symbol\": \"FUN1\" }")
+
+APP_ID_1=$(linera --with-wallet 0 project publish-and-create \
+           examples/fungible \
+           --json-argument '{ "accounts": { "User:'$OWNER_0'": "500", "User:'$OWNER_1'": "500" } }' \
+           --json-parameters "{ \"ticker_symbol\": \"FUN2\" }")
+```
+
+Each user is granted 500 tokens of each type.
+
+We also need to publish the bytecode for the Matching Engine application (without instantiating it).
+
+```bash
+ME_BCID=$(linera -w 0 publish-bytecode \
+    examples/target/wasm32-unknown-unknown/release/matching_engine_{contract,service}.wasm)
+```
+
+Lastly, we have to create the RFQ application. We pass in the bytecode ID of the Matching Engine
+bytecode, so that the application can instantiate the Matching Engine on the temporary chains for us.
+
+```bash
+APP_RFQ=$(linera -w 0 --wait-for-outgoing-messages \
+    project publish-and-create examples/rfq \
+    --json-parameters "{\"me_bytecode_id\":\"$ME_BCID\"}" \
+    --required-application-ids $APP_ID_0 $APP_ID_1)
+```
+
+We also need to make sure that both users can access the RFQ application, so we have to request it
+on the second user's chain.
+
+```bash
+linera -w 1 request-application $APP_RFQ
+```
+
+## Using the RFQ Application
+
+First, node services for the both users' wallets have to be started:
+
+```bash
+linera -w 0 service --port 8080 &
+linera -w 1 service --port 8081 &
+```
+
+### Using GraphiQL
+
+Type each of these in the GraphiQL interface and substitute the env variables with their actual
+values that we've defined above.
+
+First, user B has to claim their tokens on their chain, since they were created on user A's chain.
+
+- Claim 500 FUN1 from `$OWNER_1` in `$CHAIN_0` to `$OWNER_1` in `$CHAIN_1`, so they're in the proper chain.
+  Run `echo "http://localhost:8081/chains/$CHAIN_1/applications/$APP_ID_0"` to print the URL
+  of the GraphiQL interface for the FUN1 app. Navigate to that URL and enter:
+
+```gql,uri=http://localhost:8081/chains/$CHAIN_1/applications/$APP_ID_0
+mutation {
+  claim(
+    sourceAccount: {
+      chainId: "$CHAIN_0",
+      owner: "User:$OWNER_1",
+    }
+    amount: "500.",
+    targetAccount: {
+      chainId: "$CHAIN_1",
+      owner: "User:$OWNER_1"
+    }
+  )
+}
+```
+
+- Claim 500 FUN2 from `$OWNER_1` in `$CHAIN_0` to `$OWNER_1` in `$CHAIN_1`, so they're in the proper chain.
```

Comment:
I'd remove the dashes: Paragraphs separated by a new line won't be indented anymore, even though I think they conceptually belong to this point?